app/components/ui/Legal.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import LandingPageCard from './LandingPageCard';
+
+const HOST_QUERY = 'https://askdarcel.org/search?query=';
+const CATEGORIES_LEGAL = [
+  {
+    title: 'Housing',
+    content: 'Eviction defense, housing help, homeownership',
+    query: 'Housing+Law',
+    imgClass: 'lb-housing',
+  },
+  {
+    title: 'Immigration',
+    content: 'Asylum, SIJS, DACA, T-Visas, U-Visas, VAWA',
+    query: 'Immigration',
+    imgClass: 'lb-immigration',
+  },
+  {
+    title: 'Family & Relationships',
+    content: 'Divorce, custory, guardianship, restraining order, T-Visas, etc.',
+    query: 'Family+Law',
+    imgClass: 'lb-family',
+  },
+  {
+    title: 'Criminal & Tickets',
+    content: 'Criminal process questions, quality of life, record expungement',
+    query: 'Criminal',
+    imgClass: 'lb-criminal',
+  },
+  {
+    title: 'Work, Credit, and Consumer',
+    content: 'Credit/consumer, worker\'s rights, work and family',
+    query: 'Workers+consumers',
+    imgClass: 'lb-work',
+  },
+  {
+    title: 'Benefits',
+    content: 'CAAP, CalWORKS, CalFresh, MediCal, SSI',
+    query: 'Government+Benefits',
+    imgClass: 'lb-benefits',
+  },
+];
+
+class Legal extends Component {
+  render() {
+    return (
+      <div className="legal-block">
+        <div className="legal-block__hammer" />
+        <div className="legal-block__resources">
```

Comment:
Couldn't decide if resources should be its own block or not, but I made it a element because whatever